contacts/config/deploy.rb

```diff
@@ -30,5 +30,5 @@
   "public/templates"
 ]
 
-after "deploy:symlink", "deploy:publishing_api:publish_special_routes"
+after "deploy:symlink", "deploy:finders:publish"
```

Comment:
There's a nice gotcha here. `deploy:publishing_api:publish_special_routes` refers to the `capistrano` task, not the rake task. So `deploy:finders:publish` will fail because there's no cap task by that name.